lib/locales/ko.yml

```diff
@@ -0,0 +1,22 @@
+# :one_other pluralization (see https://github.com/ddnexus/pagy/blob/master/lib/locales/utils/p11n.rb)
```

Comment:
I think this pluralization should be `:other` and not `:one_other`

Reply:
Oh, i just skipped this line means. I'll check my fault with the other comment below and PR again. Thank your review 👍